src/views/RecIM/components/Helpers/index.js

```diff
@@ -1,11 +1,35 @@
 import { DateTime } from 'luxon';
 
 const standardDate = (date, includeTime) => {
+  if (!DateTime.isDateTime(date)) date = DateTime.fromISO(date);
```

Comment:
this looks incredibly cursed

Reply:
Is there no way to explicitly cast date to a datetime without checking if !DateTime.isDateTime(date)?